pkg/specgen/namespaces.go

```diff
@@ -96,3 +98,16 @@ func (n *Namespace) validate() error {
 	}
 	return nil
 }
+
+func NewNamespace(path string) Namespace {
```

Comment:
This probably works for Host, but won't work for `--cgroupns=container` (needs to turn into `Private`) or `--net=ns:$PATH` (needs to turn into `path:$PATH`)

Reply:
Thanks.

Reply:
Ok I moved the namespace stuff to a separate PR.